cg/cli/set.py

```diff
@@ -55,14 +54,19 @@ def family(context, action, priority, panels, family_id):
 @click.option('-s', '--sex', type=click.Choice(['male', 'female', 'unknown']))
 @click.option('-c', '--customer', help='updates customer, input format custXXX.')
 @click.option('-C', '--add-comment', 'comment', type=str, help='adds a note/comment to a sample, '
-              'put text between quotation marks! This will not overwrite the current comment.')
+                                                               'put text between quotation marks! '
+                                                               'This will not overwrite the '
+                                                               'current comment.')
 @click.option('-d', '--downsampled-to', type=int, help='sets number of downsampled \
               total reads. Enter 0 to reset.')
 @click.option('-a', '--application-tag', 'apptag', help='sets application tag.')
 @click.option('-k', '--capture-kit', help='sets capture kit.')
+@click.option('--data-analysis', help='sets data-analysis.')
 @click.argument('sample_id')
+@click.argument('user_signature')
```

Comment:
MAybe not something for now, but we can figure out who is running the command and sign the comment with the logged in user name

Reply:
That would be nice, do you know of an example where we do it like that?

Reply:
This should work

```
import getpass
username = getpass.getuser()
```

Reply:
implemented